src/components/FlawAlertsList.vue

```diff
@@ -40,6 +40,7 @@ const alertsExpanded = ref(false);
   <LabelCollapsible
     v-if="Object.values(alertsCount).reduce((acc, alertTypeCount) => acc + alertTypeCount, 0) !== 0"
     class="my-2"
+    v-bind="$attrs"
```

Comment:
is this needed? by default the root element inherits the `$attrs` unless disabled